jetty-home/src/main/resources/bin/jetty.sh

```diff
@@ -497,22 +544,23 @@ case "$ACTION" in
         su - "$JETTY_USER" $SU_SHELL -c "
           cd \"$JETTY_BASE\"
           echo ${RUN_ARGS[*]} start-log-file=\"$JETTY_START_LOG\" | xargs ${JAVA} > /dev/null &
-          disown $(pgrep -P $!)"
+          disown \$!"
```

Comment:
There is an extra closing double quote?

Reply:
This double-quote closes the one opened on line 497